backend/src/Core/FurnitureShop.Core.Contracts/Mobile/Reviews/ReviewDTOs.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace FurnitureShop.Core.Contracts.Mobile.Reviews
+{
+    public class ReviewDTO
+    {
+        public Guid Id { get; set; }
+        public ReviewInfoDTO ReviewInfo { get; set; }
```

Comment:
rename DTO object with DTO on the end of the word. It's very helpful for see difference in frontend is this object from contracts or not